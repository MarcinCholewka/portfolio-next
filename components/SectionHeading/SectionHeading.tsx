import React, { ReactNode } from 'react';

type Props = {
  children: ReactNode;
};

export const SectionHeading = ({ children }: Props) => {
  return (
    <h2 className='text-3xl text-slate-200 text-center font-medium capitalize mb-8'>{children}</h2>
  );
};
