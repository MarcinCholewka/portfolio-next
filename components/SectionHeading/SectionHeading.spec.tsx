import React from 'react';
import { render } from '@testing-library/react';

import { SectionHeading } from '@/components/SectionHeading';

import '@testing-library/jest-dom';

describe('<SectionHeading />', () => {
  it('renders correctly', () => {
    const { container, getByText } = render(<SectionHeading>Section Heading</SectionHeading>);

    expect(container).toMatchSnapshot();
    expect(getByText('Section Heading')).toBeInTheDocument();
  });
});
