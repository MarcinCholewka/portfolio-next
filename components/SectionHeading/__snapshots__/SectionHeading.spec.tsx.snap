// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionHeading /> renders correctly 1`] = `
<div>
  <h2
    class="text-3xl text-center font-medium capitalize mb-8"
  >
    Section Heading
  </h2>
</div>
`;
